### PR TITLE
When stopping service, wait until actually exited

### PR DESCRIPTION
**Pull request by otoolep**
Fixes issue #3548.

### CODE REVIEW

**Comment by otoolep** [comment]
@toddboom

**Comment by otoolep** [comment]
Tested on a DO droplet.

File: scripts/init.sh [hunk 1]
```diff
@@ -104,6 +104,20 @@ function killproc() {
...
     kill -s $3 $PID
+    while true; do
+        pidof `basename $DAEMON`
+        if [ $? -ne 0 ]; then
+            return 0
+        fi
+
+        sleep 1
+        n=$(expr $n + 1)
+        echo $n
```
Comment (jwilder) [inline scripts/init.sh]: Is this needed for anything?

Comment (otoolep) [inline scripts/init.sh]: Good man, debug -- let me remove it.

**Comment by otoolep** [comment]
Debug code removed @jwilder

**Comment by jwilder** [comment]
:+1:

**Pull request merged**